File: eTierMeeting_v2/API/Models/eTM_Team_Unit.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace eTierV2_API.Models
{
    public class eTM_Team_Unit
    {
        [Key]
        [StringLength(10)]
        public string TU_ID { get; set; }
        [Required]
        [StringLength(50)]
        public string TU_Name { get; set; }
        [Required]
        [StringLength(20)]
        public string Center_Level { get; set; }
        [Required]
        [StringLength(3)]
        public string Tier_Level { get; set; }
        [Required]
        [StringLength(8)]
        public string TU_Code { get; set; }
        [StringLength(50)]
        public string Class1_Level { get; set; }
        [StringLength(50)]
        public string Class2_Level { get; set; }
        [Required]
        [StringLength(10)]
        public string Insert_By { get; set; }
        [Column(TypeName = "datetime")]
        public DateTime Insert_Time { get; set; }
        [StringLength(10)]
        public string Update_By { get; set; }
        [Column(TypeName = "datetime")]
        public DateTime? Update_Time { get; set; }
    }
    public class eTM_Team_UnitIndexOC
    {
        public int Id { get; set; }
        public int Level { get; set; }
         public int? ParentID { get; set; }
        public string TU_ID { get; set; }
        public string TU_Name { get; set; }
        public string Center_Level { get; set; }
        public string Tier_Level { get; set; }
        public string TU_Code { get; set; }
        public string Class1_Level { get; set; }
        public string Class2_Level { get; set; }
        public int? SortSeq { get; set; }      // Sort sequence by Parent Id
        public int? LineNum { get; set; }      // The first LineNum of Child level
        public int? RowCount { get; set; }
        public string Building { get; set; }
        public bool IsActive { get; set; }
    }
}

File: eTierMeeting_v2/SPA/src/app/_core/_models/eTM_Team_Unit.ts
export interface eTM_Team_Unit {
    tU_ID: string;
    tU_Name: string;
    center_Level: string;
    tier_Level: string;
    tU_Code: string;
    class1_Level: string;
    class2_Level: string;
    insert_By: string;
    insert_Time: Date;
    update_By: string;
    update_Time: Date | null;
}
export interface eTM_Team_UnitIndexOC {
    id: number;
    level: number;
    parentID: number | null;
    tU_ID: string;
    tU_Name: string;
    center_Level: string;
    tier_Level: string;
    tU_Code: string;
    class1_Level: string;
    class2_Level: string;
    sortSeq: number | null;
    lineNum: number;
    rowCount: number | null;
    building: string;
    isActive: boolean;
}